Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseAutonomous.java

```diff
@@ -223,4 +224,25 @@ public void onError(int errorCode) {}
             return tagOfInterest.id;
         }
     }
+
+    /**
+     *EXPERIMENTAL
+     * Starts streaming a camera and sets a pipeline to it, this is a method now because it will have to be called multiple times in any auton class where pipeline switching is used
+     * @param pipeline pipeline that the method inputs, any pipeline-specific methods will have to be called separately
+     * @param camera camera that the method inputs
+     * @param X desired camera width (in pixels)
+     * @param Y desired camera height (in pixels)
+     */
+    public void startCameraWithPipeline(OpenCvPipeline pipeline, OpenCvCamera camera, int X, int Y){
```

Comment:
```suggestion
    public void startCameraWithPipeline(OpenCvPipeline pipeline, OpenCvCamera camera, int X, int Y) {
```

Reply:
There was a weird thing where I had to stash changes and push, it wouldn't let me resolve merge conflicts. That is likely why the stuff got set back to before